index.html

```diff
@@ -0,0 +1,121 @@
+                    </li>
+    
+                </ul>
+            </div>
+
+            <div id="github_image">
+                
+                <a target="_blank" id="profile-link" href="https://github.com/matovu-farid/microverse_maxwell_farid/blob/85fcd846190f554111ebfb836d051ce16a43092b/index.html">
+                    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Octicons-mark-github.svg/1200px-Octicons-mark-github.svg.png" alt="Github">
+                    Github
+                </a>
+            </div>
+        </nav>
+    </header>
+    <section id="welcome-section">
+        <h1>
+            Welcome to my Portfolio
+        </h1>
+
+    </section>
+    <section id="projects">
+        <div>
+            <h1>Projects</h1>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut possimus saepe ullam ipsa. Repellendus officiis exercitationem dolor nobis ea voluptatibus obcaecati, sit minus. Placeat, ut perspiciatis maxime aliquam iste rem.
+        </div>
+
+      <div class="project-tile">
+        <h2 class="project-title">
+            Restaurant Website
+        </h2>
+        <img src="https://imaxxiportfolio.netlify.app/assets/restau.JPG" class="restau-imgs" alt="project">
+        <img src="https://imaxxiportfolio.netlify.app/assets/restau_2.JPG" class="restau-imgs" alt="project">
+        
+        <p class="description">
+            A restaurant advertisement and order placement website. 
+            Come one, come all ! Get access to special delicasies with ease.
+            <br>
+            <a href="https://restaurantconfusion.netlify.app">Check it out</a>
+            
+            
+        </p>
+      </div>
+
+       
+      <div class="project-tile">
+      
+
+        <h2 class="project-title">
+            Painter App
+        </h2>
+        <img src="https://farid-website.firebaseapp.com/assets/painter.png" class="phone-img" alt="project">
+        
+        <p class="description">
+            This is an app that enables anyone to draw. It is equipped with a number of features that make drawing easy. It also enables saving of your drawing.
+            <br>
+            
+                <a class="button" href="https://play.google.com/store/apps/details?id=com.faridmatovu.painter_app" target="_blank" rel="noopener noreferrer">
+
+                    Install Now
+                </a>
+           
+            
+        </p>
+      </div>
+    </section>
+
+    <section id="skills">
+        <h2>Skills</h2>
+        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quam
+        adipisci reprehenderit beatae inventore odio? Suscipit, provident
+        odio eveniet odit enim recusandae aperiam corporis, nobis quasi
+        deleniti adipisci molestias nisi aliquam.
+    </section>
+    <section id= "contacts">
+        <div class="" id="contact">
+            <h2 class="">Contact Me</h2>
+
+        
+            <div class="w3-section">
```

Comment:
Please replace the div element with an HTML 5 semantic element like the section element. A semantic element clearly describes its meaning to both the browser and the developer. This will make it easier for search engines as well as make your code more readable.